### ISSUE TITLE
Tree keyboard navigation

### ISSUE THREAD

**Issue by evansjohnson** [palantir/blueprint#6860]
#### Code Sandbox
https://blueprintjs.com/docs/#core/components/tree

#### Steps to reproduce
Try to use keyboard to interact with the tree, nothing is focusable

#### Actual behavior
Not able to focus and navigate around Tree component

#### Expected behavior
Should be able to use keyboard nav to interact with and navigate around Tree

#### Possible solution
Add keyboard nav, following https://www.w3.org/WAI/ARIA/apg/patterns/treeview/examples/treeview-navigation/ or similar
